js/libs/keycloak-admin-client/src/resources/serverInfo.ts

```diff
@@ -14,4 +15,23 @@ export class ServerInfo extends Resource {
     method: "GET",
     path: "/",
   });
+
+  public findEffectiveMessageBundles = this.makeRequest<
+    {
+      realm: string;
+      theme?: string;
+      themeType?: string;
+      local?: string;
+      hasWords?: string;
```

Comment:
This param is never used, so it can likely be removed. Or it should be added to one of the params.
```suggestion
```

Reply:
Thanks @jonkoops Let's keep it as It will be used in the filter form in the FE. Erik will add it to his java endpoint implementation.

Reply:
The client should always reflect the current status of the API, so I'd prefer it if we didn't add this now, even if it might be added relatively soon. We're also about to do a release, so this might slip through the cracks.

If Erik raises a PR to implement this we can add it under that one.

Reply:
Thanks, sounds good to me!